ts/components/MemberListItem.tsx

```diff
@@ -20,12 +18,60 @@ const AvatarItem = (props: { memberPubkey: string; isAdmin: boolean }) => {
   );
 };
 
+const StyledSessionMemberItem = styled.button<{
+  inMentions?: boolean;
+  zombie?: boolean;
+  selected?: boolean;
+}>`
+  cursor: pointer;
+  flex-shrink: 0;
+  flex-grow: 1;
+  font-family: var(--font-default);
+  padding: 0px var(--margins-sm);
+  height: ${props => (props.inMentions ? '40px' : '50px')};
+  display: flex;
+
```

Comment:
can we remove these new lines between the flex properties?

Reply:
updated

Reply:
👍